api/sql/schema/750-rule.decision.fetch.sql

```diff
@@ -256,7 +261,7 @@ BEGIN TRY
                 r.startAmountDaily DESC,
                 r.startAmount DESC,
                 r.splitRangeId),
-            RANK() OVER (ORDER BY c.priority, c.conditionId)
+            RANK() OVER (ORDER BY c.priority, c.name, c.conditionId)
```

Comment:
The priority is not required to be unique, so the ordering is also by name, for the conditions with the same priority.